support-frontend/assets/pages/paper-subscription-checkout/helpers/paymentProviders.js

```diff
@@ -1,34 +1,42 @@
 // @flow
 
-import { loadStripe, openDialogBox, setupStripeCheckout } from 'helpers/paymentIntegrations/stripeCheckout';
-import { type IsoCountry } from 'helpers/internationalisation/country';
 import {
-  type PaymentAuthorisation,
-  type PaymentResult,
-  postRegularPaymentRequest,
   regularPaymentFieldsFromAuthorisation,
-  type RegularPaymentRequest,
 } from 'helpers/paymentIntegrations/readerRevenueApis';
-import { type Dispatch } from 'redux';
-import { openDirectDebitPopUp } from 'components/directDebit/directDebitActions';
-import { finalPrice as paperFinalPrice } from 'helpers/productPrice/paperProductPrices';
-import { routes } from 'helpers/routes';
 import { getQueryParameter } from 'helpers/url';
 import { getOphanIds, getSupportAbTests } from 'helpers/tracking/acquisitions';
 import { Monthly } from 'helpers/billingPeriods';
 import { type FormFields, getFormFields } from 'components/subscriptionCheckouts/address/addressFieldsStore';
 
 import {
   type Action,
-  getBillingAddress,
-  getDeliveryAddress,
   setFormSubmitted,
   setStage,
   setSubmissionError,
   type State,
 } from '../paperSubscriptionCheckoutReducer';
 import { DirectDebit, Stripe } from 'helpers/paymentMethods';
 
+import {
+  getBillingAddress,
+  getDeliveryAddress,
+} from '../paperSubscriptionCheckoutReducer';
+import type {IsoCurrency} from "../../../helpers/internationalisation/currency";
```

Comment:
This can just be `helpers/internationalisation/currency`

You can get Intellij to do this by default by checking the 'Use paths relative to project, resource or sources roots' checkbox in Settings | Editor | Code Style | JavaScript | Imports

Reply:
now this is the kind of comment I like in my PRs! 💪 power up!

Reply:
erm actually this file the imports are really messed up! I'll tidy.  Do we have a specific ordering e.g. alphabetical?

Reply:
Not really. As I say I would favour imports relative to the assets directory, which reminds me that you probably want to mark `assets` as a source root in intellij or it will get confused.

Reply:
wow, this is gold, thanks, I was getting annoyed about all the imports being underlined! 🌟